.github/actions/ginkgo/main-focus.yaml

```diff
@@ -187,7 +187,7 @@ include:
     cliFocus: 'K8sDatapathServicesTest Checks E/W loadbalancing \\(ClusterIP, NodePort from inside cluster, etc\\) TFTP|K8sDatapathServicesTest Checks E/W loadbalancing \\(ClusterIP, NodePort from inside cluster, etc\\) with'
 
   ###
-  # K8sDatapathServicesTest Checks E/W loadbalancing (ClusterIP, NodePort from inside cluster, etc) Tests NodePort inside cluster (kube-proxy) 
```

Comment:
oh, this space is needed, that's how the test is called.

Reply:
Oh good catch. My editor prunes extra whitespace on save so I didn't intend to do this.